packages/universal-components/src/Icon/icons.json

```diff
@@ -1187,5 +1187,55 @@
     "description": "",
     "character": "E171",
     "added": "0.28"
+  },
+  "upload" : {
```

Comment:
Thanks, but we will also need to obtain new `orbit-icons.ttf` because new icons are not included in current one. If it's not inside Orbit repository, maybe try to ask in `#plz-orbit` channel.

<img width="426" alt="Screenshot 2019-03-21 at 11 55 38" src="https://user-images.githubusercontent.com/2660330/54748188-39b63000-4bd1-11e9-909d-64ba96ba2e74.png">

Reply:
Yes! They will upload them today at the repo and then we can add them here 👍

Reply:
@JoseDuSV New fonts added!

Reply:
Thanks, that was quick. Can you please also update it inside `apps/web/static/fonts` folder.
Then just squash the commits and we can merge it.

Reply:
@JoseDuSV Done!